src/index.js

```diff
@@ -341,7 +342,6 @@ function extractRawLocation(item) {
     currentlyOpenForReceiving: item.gsx$currentlyopenforreceiving.$t,
     aidReceivingHours: getHours(item.gsx$openingforreceivingdonations.$t, item.gsx$closingforreceivingdonations.$t),
     seekingVolunteers: item.gsx$seekingvolunteers.$t,
-    urgentNeed: item.gsx$urgentneed.$t,
     notes: item.gsx$notes.$t
   }
```

Comment:
The order of things in this object literal are the order things show up in the pop up. This is non-obvious and a side effect of not having a really templating language to describe html fragments.